marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts

```diff
@@ -130,7 +131,7 @@ export class ProductDetailComponent {
     this.updateDropdownSelection();
   }
 
-  constructor() {
+  constructor(private readonly titleService: Title) {
```

Comment:
I think you missed this case 

![image](https://github.com/user-attachments/assets/05c7e4c1-f97f-41ee-bf3b-1db0f343329f)

Reply:
This case is not a part of this story.
Could you create another various task to handle it?